src/locales/en.json

```
@@ -333,10 +333,10 @@
     },
     "lengths": {
       "title": "Durations",
-      "shortest": "Shortest",
-      "short": "Short",
-      "medium": "Medium",
-      "long": "Long"
+      "shortest": "Shortest (< 30s)",
```

```suggestion
      "shortest": "Shortest (<30s)",
```

For consistency with the 'long' label